lib/grid_game.js

```diff
@@ -62,6 +63,12 @@ var GridGame = {
     this.keyboard = new GridGame.classes.keyboard();
     this.keyboard.init();
   },
+
+  init_obstacles: function() {
+    for(var i=0; i < this.rock_count; i++){
```

Comment:
Another small point about optimizations (since we're talking about games);

When you're using a loop like `for`, there's a strange quirk that if you work from the _end_ of the array _backwards_ you can actually speed up the loop by a non-insignificant factor.  

If you want to find out more about why this is (and why it's important) you can read this thread: http://stackoverflow.com/questions/8689573/why-is-iterating-through-an-array-backwards-faster-then-forwards